index.html

```diff
@@ -8,22 +8,24 @@
   <body>
     <header>
       <nav>
-        <div id="logo">Logo</div>
-        <div>Nav Item 1</div>
-        <div>Nav Item 2</div>
-        <div>Nav Item 3</div>
-        <div>Nav Item 4</div>
+        <section>
+          <img id="logo" src="http://via.placeholder.com/200x70/000000" />
+          <div>Nav Item 1</div>
```

Comment:
When creating future apps, you should put your navigation items in a list. This is important for web accessibility as screen readers provide extra information to the users when things are in lists.